core/src/main/java/org/mule/runtime/core/MuleServer.java

```diff
@@ -61,7 +61,7 @@
     /**
      * Don't use a class object so the core doesn't depend on mule-module-spring-config.
      */
-    protected static final String CLASSNAME_DEFAULT_CONFIG_BUILDER = "org.mule.runtime.core.config.builders.AutoConfigurationBuilder";
+    public static final String CLASSNAME_DEFAULT_CONFIG_BUILDER = "org.mule.runtime.core.config.builders.AutoConfigurationBuilder";
```

Comment:
Perhaps we should add a TODO in order to revert this and find a better way to write the test that now it is in another package.